src/Controller/PortalController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of aakb/jira_economics.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/portal", name="portal_")
+ */
+class PortalController extends Controller
```

Comment:
Should probably be changed after merging of https://github.com/aakb/jira_economics/pull/18